fastlane/lib/fastlane/actions/update_fastlane.rb

```diff
@@ -55,6 +55,12 @@ def self.run(options)
           return
         end
 
+        unless updater.respond_to? :highest_installed_gems
+          Helper.log.warn "The update_fastlane action requires rubygems version 2.1.0 or greater."
+          Helper.log.warn "Please update your version of ruby gems before proceeding."
```

Comment:
Maybe we should tell users how to do so at this point?

```sh
gem install rubygems-update
update_rubygems
gem update --system
```

Could you replace `Helper.log.warn` with

```
UI.important "Text here"
...
UI.command "gem update --system"
...
```